SUI/SUI/ContentView.swift

```diff
@@ -4,15 +4,172 @@
+        static let windowString = "Тонированная лобовуха:"
+        static let spoilerString = "Спойлер:"
+        static let pledgeString = "Залог:"
+        static let rublesString = "рубасов"
+        static let imageWidth:CGFloat = 300
+        static let imageHeight:CGFloat = 240
+        static let cornerRadius:CGFloat = 10
+        static let offsetXNumber = -500
+        static let timeNumber = 0.5
+        static let zeroNumber = 0
+        static let textSizeNumber: CGFloat = 15
+        static let zeroInt = 0
+        static let zeroFloat: Float = 0
+        static let zeroDouble: Double = 0
+    }
+    
+    @State var segmentIndex = Constants.zeroInt
+    @State var vEngineIndex = Constants.zeroInt
+    @State var pickerIndex: Float = Constants.zeroFloat
+    @State var carCost = Constants.carCostsValues
+    @State var firstPrice: Double = Constants.zeroDouble
+    @State var offsetX = Constants.zeroInt
+    @State var isOnEngine = false
+    @State var isCall = false
+    @State var isMyChoose = false
+    @State private var isEditingSlider = false
+    var cars = Constants.carsValues
+    var typeOfSpoiler = Constants.typeOfSpoilersValues
+    
+    @State private var isSharePresented = false
+    let customActivity = CustomActivityView(title: Constants.socialString, imageName: Constants.activityImageName) {
+    }
+    
+    private var carImage: some View {
+        Image(cars[segmentIndex])
+            .resizable()
+            .frame(width: Constants.imageWidth, height: Constants.imageHeight)
+            .offset(x: CGFloat(offsetX))
+            .cornerRadius(Constants.cornerRadius)
+    }
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            NavigationView {
+                Form {
+                    Section {
+                        VStack {
+                            ZStack {
+                                RoundedRectangle(cornerRadius: Constants.cornerRadius)
+                                    .fill(.gray)
+                                    .offset(x: CGFloat(offsetX))
+                                    .padding()
+                                carImage
+                            }.animation(.spring())
+                            
+                            Picker(selection: Binding(get: {
+                                self.segmentIndex
+                            }, set: { newValue in
+                                self.segmentIndex = newValue
+                                self.offsetX = Constants.offsetXNumber
+                                self.moveBack()
+                            }), label: Text("")) {
+                                ForEach(0 ..< cars.count) {
+                                    Text(self.cars[$0]).tag($0)
+                                }
+                            }
```

Comment:
```suggestion
                            Picker(selection: Binding(get: {
                                self.segmentIndex
                            }, set: { newValue in
                                self.segmentIndex = newValue
                                self.offsetX = Constants.offsetXNumber
                                self.moveBack()
                            }), label: Text("")) {
                                ForEach(0 ..< cars.count) {
                                    Text(self.cars[$0]).tag($0)
                                }
                            }
```вынеси Пикер в вычисляемое свойство